### PR TITLE
Use chunked entrypoints

### PR DESCRIPTION
**Pull request by mariusandra**
## Changes

- Instead of `index.js`, we now have `index.js` AND `index-HASH.js` (same for `.css`)
- The generated `.html` files (to be further consumed by django) will contain the correct HASH inside.
- Code smell: I hardcoded a bunch of paths, but since there are already other hardcoded paths in these scripts, I'm personally fine with this.

## How did you test this code?

- Reloaded the browser locally, before the build had finished (got `index.js?t=` still), and also after (got `index-HASH.js`), and saw that the right files were loading.
image
- Made sure the toolbar loaded fine
- Also checked saved dashboards
image

### CODE REVIEW

**Comment by guidoiaquinti** [comment]
LGTM but I'm a frontend/JS newbie so I appreciate another pair of 👀

File: frontend/utils.mjs [hunk 1]
```diff
@@ -75,10 +81,16 @@ export function copyIndexHtml(from = 'src/index.html', to = 'dist/index.html', e
                             } 
                         } 
                     }
-                    window.ESBUILD_LOAD_SCRIPT("${entry}.js?t=" + new Date().valueOf())
+                    window.ESBUILD_LOAD_SCRIPT(${
+                        entrypoints.length > 0
```
Comment (macobo) [inline frontend/utils.mjs:85]: Nit: Let's extract this to a var and explicitly mention there can only be 1 js/css endpoint if passed an array. IT's not really obvious as-is

File: frontend/utils.mjs [hunk 2]
```diff
@@ -188,7 +202,31 @@ export async function buildOrWatch(config) {
             }
         }
         inputFiles = getInputFiles(result)
-        return getChunks(result)
+
+        const entrypoints = []
+
+        if (config.outdir) {
```
Comment (macobo) [inline frontend/utils.mjs:208]: Nit: Feels like this wants to get extracted plus added context.

**Review by macobo — APPROVED** [review]
I think I understand ~90% of the data flow here and it makes sense, though it does add a ton of complexity.

**Comment by mariusandra** [comment]
I hope the latest changes improved the data flow. Just in case, requesting a new review.

**Review by macobo — APPROVED** [review]
Looking even better :)

**Pull request merged**